sovrin_client/test/agent/conftest.py

```diff
@@ -413,7 +413,7 @@ def chk():
         assert link.remoteIdentifier == inviteeAcceptanceId
         assert link.remoteEndPoint[1] == inviteeAgent.endpoint.ha[1]
 
-    emptyLooper.run(eventually(chk, timeout=10, retryWait=0.2))
+    emptyLooper.run(eventually(chk))
```

Comment:
You are decreasing the timeout here from 10 to 5 seconds, sure it will not fail?